spec/lib/rex/proto/kerberos/pac/krb5_pac_spec.rb

```diff
@@ -112,6 +134,17 @@
     end
   end
 
+  describe '#write' do
+    it 'writes then reads back to its original state' do
+      pac.assign(pac_elements: pac_elements_with_upn)
+      pac.sign!
+      data = pac.to_binary_s
+      print("data is #{data.inspect}\n")
```

Comment:
```suggestion
```

Funny timing; a contributor spotted some left behind logging here too https://github.com/rapid7/metasploit-framework/issues/18562 😄

Reply:
Totally irrelevant comment, I like using the pattern `$stderr.puts("data is #{data.inspect}\n")` as it's less likely that the pattern would be shadowed, i.e. a `def print` or `def puts` already being in the current scope

Reply:
Good call - thanks @adfoster-r7